docs/Development-Guides/Data-Syncs.md

```diff
@@ -0,0 +1,17 @@
+---
+stoplight-id: f7af2039f9a7b
+---
+
+# Data Syncs
+
+Finch syncs data with providers on a 24 hour cadence, and Finch API responses will return data from the most recent successful sync. You can use several Finch response headers to determine the freshness of the data you receive:
+
+Response Header | Description
+-------|--------------
+`Finch-Data-Retrieved` |	The date/time that the data was retrieved from the employment system.
+`Finch-Last-Attempted-Update-Date` | The date/time that Finch last attempted to sync the data for this endpoint.
+`Finch-Last-Attempted-Update-Result` | The result of Finch’s last attempted update. Possible values:<br />`success`: The last attempt succeeded<br />`error`: The last attempt errored<br />`partial_error`: (batch requests only) The last attempt succeeded for some items in the batch and failed for others.
+
+For batch endpoints (`/employment`, /`individual`, `/pay-statement`) Finch syncs all of the data for the connection in one atomic operation. This ensures that all items in the batch response you receive are from the same data sync.
+
+Performing data syncs in this way ensures that latency for all Finch API requests should remain under 200ms.
```

Comment:
I can see customers misunderstanding "all Finch API requests".

**Suggested:**
Performing data syncs in this way ensures that latency for the most common Finch API requests should remain under 200ms. API requests for partial or specific data are expected to experience higher latencies.

Reply:
I don't think that's true though, which would you think would have higher latencies?

Reply:
@alexromano If I call Payment and ask for the last 2 years, it should (in theory) be 200ms or less because we already synced and cached that response. If I ask for the last 1 year, however, that's a request for partial data (compared to our cache) and I would expect that we make a live call. (And, because it's live, I would expect it to have higher latency than our cache.) Is that incorrect?

Reply:
No live call will be made. We will just slice the payments data we store and return only the 1 year worth

Reply:
@alexromano Based on Slack DMs, revising my suggestion:

Performing data syncs in this way ensures that latency for most Finch API requests should remain under 200ms. API request latency for data which Finch has not been able to cache, such as first connections, would be higher.